.readthedocs.yml

```
@@ -1,12 +1,16 @@
 ---
 
 build:
-  image: latest
+  os: ubuntu-22.04
+  tools:
+    python: "3.6"
```

My personal preference is to avoid quotes whenever possible. So I tend to do this:
```suggestion
    python: >-
      3.6
```
(it's a nice practice that if used consistently, helps avoid accidental problems with having to escape stuff)

---

I'm pretty sure I tried without quotes and RTD didn't accept it as a valid value. Would you like me to try again?

---

Just apply this code suggestion using the GH UI button. It'll work. The `>-` syntax produces strings, just like inline quotes. But without it, the YAML parser would see a float value which is the problem you were seeing earlier. RTD disallows floats and fails, but with an explicit string, it's fine.